src/main/_docs/setup/setup-managing.md

```diff
@@ -80,9 +80,10 @@ The upgrade process:
 5. Backs up your installation
 6. Initializes the new version
 7. Starts Che
+8. Important! If `CHE_PREDEFINED_STACKS_RELOAD__ON__START` is set to false, stacks packaged into new binaries won't be saved into a database.
```

Comment:
Need to describe the property in the "configuration.md" page as it explains most of the `che.env` properties and then link to the explanation from here.